### DISCUSSION TITLE
GPT Assistant interface?

### DISCUSSION THREAD

**Discussion by richardwhite00** [lspahija/AIUI#44] [Ideas]
Hi there! Your project is awesome and I managed to get it to work but how hard would it be to adapt it now to the new GPT Assistant API?

**Comment by Braunfeltd** [comment]
Its not hard, you will just need to update the ai.py file to use the new models and build what ever tool functions in to support your inputs.

**Comment by richardwhite00** [comment]
You're a rock star! Thanks so much for responding! I'll get to work on it now. Sent from my T-Mobile 4G LTE Device
-------- Original message --------From: Braunfeltd ***@***.***> Date: 11/22/23  2:03 PM  (GMT-05:00) To: lspahija/AIUI ***@***.***> Cc: richardwhite00 ***@***.***>, Author ***@***.***> Subject: Re: [lspahija/AIUI] GPT Assistant interface? (Discussion #44) 
Its not hard, you will just need to update the ai.py file to use the new models and build what ever tool functions in to support your inputs.
